src/control/server/ctl_ranks_rpc_test.go

```diff
@@ -258,16 +244,16 @@ func TestServer_CtlSvc_PrepShutdownRanks(t *testing.T) {
 
 func TestServer_CtlSvc_StopRanks(t *testing.T) {
 	for name, tc := range map[string]struct {
-		missingSB        bool
-		engineCount      int
-		instancesStopped bool
-		req              *ctlpb.RanksReq
-		signal           os.Signal
-		signalErr        error
-		ctxTimeout       time.Duration
-		expSignalsSent   map[uint32]os.Signal
-		expResults       []*sharedpb.RankResult
-		expErr           error
+		missingSB         bool
+		engineCount       int
+		instancesStopped  bool
+		instancesDontStop bool
```

Comment:
I would think if one of these was false, the opposite would be true...

Reply:
the first indicates that the rank is already stopped, the second indicates that the rank refuses to stop, they are different cases